Tasks/AzureRmWebAppDeployment/task.json

```diff
@@ -116,6 +123,15 @@
             "required": false,
             "defaultValue": "",
             "helpMarkDown": "Additional Web Deploy arguments that will be applied when deploying the Azure Web App like,-disableLink:AppPoolExtension -disableLink:ContentExtension."
+        },
+		{
+            "name": "outputVariable",
```

Comment:
can you name it webappuri instead of outputVariable. We may introduce more output tomorrow and it will make sense to keep this variable name relevant.